[Qumulo.Storage] APEX - Create a new module: Qumulo, and support it auto generated (#22152)
Azure/azure-powershell: src/Qumulo/examples/Remove-AzQumuloFileSystem.md

> ### Example 1: Remove special resource with sepecial group

Reviewer: ```suggestion
### Example 1: Remove specific resource with specified resource group
```

```powershell
Remove-AzQumuloFileSystem -Name qumulo01 -ResourceGroupName ps-joyer-test02
```

> Remove special File System Resource in sepecial group

Reviewer: ```suggestion
Remove specific file system resource in specified resource group
```

> ### Example 2: Get and remove special resource with sepecial group

Reviewer: ```suggestion
### Example 2: Get and remove specific resource with specified resource group
```

```powershell
$fileSystem = Get-AzQumuloFileSystem -ResourceGroupName ps-joyer-test -Name qumulo-resource-01
Remove-AzQumuloFileSystem -InputObject $fileSystem
```

> Remove special File System Resource in sepecial group

Reviewer: ```suggestion
Remove specific file system resource in specified resource group
```
